qemu/tests/virtio_fs_write_same_space.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+from virttest import data_dir
+from virttest import env_process
+from virttest import error_context
+from virttest import nfs
+from virttest import utils_disk
+from virttest import utils_misc
+from virttest.remote import scp_to_remote
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Write to the same space test on shared directory.
+    Steps:
+    1. setup nfs if needed
+    2. boot up guest with virtiofs device
+    3. mount virtiofs on guest
+    4. with mmap to write to the same space on shared dir
+    5. check the size of the file
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    test_file = params.get('test_file')
+    fs_dest = params.get('fs_dest')
+    fs_target = params.get("fs_target")
+    script_create_file = params.get("script_create_file")
+    cmd_create_file = params.get("cmd_create_file")
+    username = params.get('username')
+    password = params.get('password')
+    port = params.get('file_transfer_port')
+    setup_local_nfs = params.get('setup_local_nfs', "no")
+
+    try:
+        vm = None
```

Comment:
Suggest remove `vm = None`

Reply:
We need it in case  unexpected error during setup_local_nfs, and then L74  compained there is no vm defined.

Reply:
Okay,got it.
LGTM.